### DISCUSSION TITLE
Firefox quantum é tão bom assim?

### DISCUSSION THREAD

**Discussion by romulosardinha** [frontendbr/forum#819] [Discussão]
Alguém aí tá usando o ff quantum? Eu não senti diferença aqui em comparação com o chrome.

E vcs ? Qual usam?

**Comment by wmsbill** [comment]
Firefox e Ele é muito rápido. Digamos que o FF é rápido no sentido de que você tem performance mais perto de 60fps. E como eles conseguem isso? Paralelismo.

**Comment by TiagoDanin** [comment]
Tentei usar, não durei uma semana. Ainda tá muito lento, Vivaldi, Opera e Chrome ainda são mais rápidos.

**Comment by robsonbittencourt** [comment]
Usei por uma semana no beta e estava gostando.
Mas achei a versão para Androi muito zuada. Acabei voltando para o Chrome para usar o mesmo nas duas plataformas.

**Comment by romulosardinha** [comment]
Ainda não consegui ganho de produtividade com ele, ainda performo melhor no chrome.

**Comment by MendesGabriel** [comment]
Ele tem uma cagada de UX no mac, as abas são cheias e você não consegue arrastar o navegador pra outras telas facilmente, você precisa achar um espacinho sem aba pra arrastar, já me atrapalhou algumas vezes isso. ( É só um comentário, eu uso pra acessar gmail e outras coisas da empresa e aqui é rápido igual o chrome. )

**Comment by waldemirflj** [comment]
@rmlSardinha, eu estou gostando... Vai fazer 1 semana que estou utilizando, e acho que vou migrar. No começo você acha estranho e até vai fazer compações idiotas, até aí normal, trata-se de um outro browser. 

@MendesGabriel, cara no meu Mac Pro Retina e Dell isso não acontece, acho que você teve azar hein. hehehe

Resumindo, esse FF é muito rápido e nem um pouco fominha comparado com o Chrome.

**Comment by MendesGabriel** [comment]
<img width="938" alt="screen shot 2017-11-20 at 5 53 27 pm" src="https://user-images.githubusercontent.com/5855697/33038465-e2199262-ce1b-11e7-94c4-ffb3d1dc0e87.png">

Só sobra um cantinho na esquerda e na direita pra jogar o navegador pra outro lado.

**Comment by mahenrique94** [comment]
Ainda prefiro o Chrome

**Comment by aferreira44** [comment]
Brave (https://brave.com/) é o navegador mais rápido que já testei até hoje. Criado pelo Brendan Eich (pai do JavaScript), o Brave também bloqueia trackers e anúncios nativamente.

Além disso, tem um sistema integrado de pagamentos em Bitcoin para produtores de conteúdo e usuários que visualizam ads da rede de anuncinates do Brave, que também podem ser desativados.

Vale a pena conferir.

**Comment by munizart** [comment]
Sim.

**Comment by giovannism20** [comment]
@aferreira44 Ele também funciona no iOS && no Android ;) .

**Comment by felipeuntill** [comment]
Testei e achei rápido como um foguete mais não consegui substituir o chrome pelo habito de uso do devtools.

**Comment by giovannism20** [comment]
@felipeuntill Já testou o Firefox Developer Edition ?

**Comment by giovannism20** [comment]
Acho que um grande problema no Firefox, é o DRM que mesmo ativado, ainda não consegui fazer funcionar, assim como não funciona "live" no YouTube...

**Comment by fxcosta** [comment]
Eu estou usando ele fulltime por enquanto, tanto no Win32 quanto no Linux e ele me atraiu em muitos poucos mas tem seus contras. O design apesar de estar bonito tem umas coisas zoadas no Linux. Em questão de performance, nos primeiros dias eu percebi que ele realmente era mais rápido que o Chrome mas não sei porque ultimamente tenho duvidado dele nesse quesito. Preciso fazer mais testes pra saber se é a rede ou se é ele de fato. Já o developer tools dele está bem legal mas nada revolucionário... nesse ponto o Chrome ainda ganha, na minha opinião. Acredito que tem tudo pra melhorar cada vez mais.

**Comment by fabiomirandaa** [comment]
Eu só achei a UI dele péssima. Mas, parece estar ,muito bom em questão de desempenho.

**Comment by expalmer** [comment]
Já tentei usar o firefox, a performance está boa, mas a UI dá dor de cabeça, muito pesada, poderiam fazer algo mais clean, suave.

**Comment by giovannism20** [comment]
@fabiomirandaa @expalmer | O que vocês mudariam ?

**Comment by expalmer** [comment]
@GiovanniSM20 tipo o chrome e o safari, são tão elegantes, desde as cores, as bordas...tipo você pega o iTerm2 e o Hyper, o Hyper é a mesma coisa só que a UI dele é clean e principalmente minimalistas.
O Firefox tem cores fortes, pesadas, hora as coisas são arredondadas e hora quadradas, o devtools tem uma fonte feia...eu não sou um especialista, mas a questão é que eu começo a usar e me canso, coisa que não acontece com o safari ou chrome...só não vou pro safari porque acho o devtools dele confuso.

**Comment by leandroruel** [comment]
A pegada do Quantum nao é nem tanto em carregamento de página, porque, nessa questão 99% do trabalho é da sua velocidade de conexão, mas no consumo de memória. menos memória = menos travamentos

**Comment by cleitonalbino** [comment]
Eu achei mais rápido, uso o FF como navegador padrão faz alguns bons anos, o único ponto negativo é que muitos add-ons pararam de funcionar por falta de compatibilidade!

**Comment by nestorjunior** [comment]
Sobre o FF, realmente (não sei se por estar usando o Chrome a muito tempo) parece que é difícil de re-acostumar com ele (essa questão da UI no Mac é bem chato).
 
Pois é @aferreira44 o único problema do Brave, é a questão do Devtools dele, que ainda está abrindo em janela separada, acho um pouco incomodo trabalhar dessa forma. Outro que também tinha esse problema era o Vivaldi mas andaram ajustando isso.
